src/astro/sql/operators/agnostic_stats_check.py

```diff
@@ -230,7 +228,19 @@ def handler_func(results):
     def execute(self, context: Dict):
         conn_type = BaseHook.get_connection(self.conn_id).conn_type  # type: ignore
         checkHandler = ChecksHandler(self.checks, conn_type)
-        metadata = MetaData()
+
+        valid_db = {
+            "postgres": Database.POSTGRES,
+            "postgresql": Database.POSTGRES,
+            "bigquery": Database.BIGQUERY,
+            "google_cloud_platform": Database.BIGQUERY,
+        }
+
+        metadata_params = {}
```

Comment:
An alternative to line 229 and these would be to capture the DB using:
```
from astro.utils.database import get_database_from_conn_id
database = get_database_from_conn_id(self.conn_id)
```

Reply:
Done.